library/patient.inc

```diff
@@ -1813,3 +1813,14 @@ function is_patient_deceased($pid, $date = '')
         return $results;
     }
 }
+
+function getPriorAuth($pid)
+{
+    $sql = "select prior_auth_number from form_prior_auth where pid = ?";
```

Comment:
hi @juggernautsei, what if there are multiple?

Reply:
I know that the way the current form is structured there could be multiple forms with multiple numbers. I think I am going to tank this commit until I can finish the form build that I am working on. This will make more since then. Because there will be only one form entry and the system will open that same form every time.